sonar-html-plugin/src/test/resources/checks/LangAttributeCheck.html

```diff
@@ -15,3 +15,23 @@
 </html>
 
 <html  <?php language_attributes(); ?>> <head></head> </html> <!-- Compliant -->
+
+<html th:lang="${#locale.language}" > <!-- Compliant -->
+<head></head>
+</html>
+
+<html th:xmllang="${#locale.language}" > <!-- Compliant -->
+<head></head>
+</html>
+
+<html th:lang-xmllang="${#locale.language}" > <!-- Compliant -->
+<head></head>
+</html>
+
+<html th:attr="lang="en"" > <!-- Compliant -->
+<head></head>
+</html>
+
+<html th:attr="xml:lang="en"" > <!-- Compliant -->
+<head></head>
+</html>
```

Comment:
I think it's worth adding the False Negative code examples mentioned before here as well, just to document them :)

Reply:
I add one False positive case. Let me know if you think about other potential false positives.